src/main/java/mServer/crawler/sender/MediathekArd.java

```diff
@@ -379,8 +380,8 @@ private void filmSuchen2(final String urlFilm_, final String thema, final String
         }
         String subtitle = seite2.extract("subtitleUrl\":\"", "\"");
         if (!subtitle.isEmpty()) {
-          if (!subtitle.startsWith("http://www.ardmediathek.de")) {
-            subtitle = "http://www.ardmediathek.de" + subtitle;
+          if (!subtitle.startsWith("http")) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1066)